[iOS] Image wrongly layouted when a transform is applied (#2803)
unoplatform/uno: doc/ReleaseNotes/_ReleaseNotes.md

### Bug fixes

> - [iOS] Applying a `<RenderTransform>` on an image were producing wrong layout result.

Reviewer: ```suggestion
- [iOS] Applying a `<RenderTransform>` on an image was producing an incorrect layout result.
```
